### PR TITLE
Add SubtleAlert UI elements to HMI guides

### PR DESCRIPTION
**Pull request by jacobkeeler**
Core Guide updates to accompany https://github.com/smartdevicelink/sdl_core/pull/3459

### CODE REVIEW

File: docs/Integrating Your HMI/Developing the HMI UI/index.md [hunk 1]
```diff
@@ -167,6 +167,12 @@ There are several RPCs which are used to display a popup or an overlay to the us
 
 ![Alert](./assets/alert.png)
 
+### UI.Alert
```
Comment (ShobhitAd) [inline docs/Integrating Your HMI/Developing the HMI UI/index.md:170]:
```suggestion
### UI.SubtleAlert
```

File: docs/Integrating Your HMI/Developing the HMI UI/index.md [hunk 2]
```diff
@@ -294,6 +300,18 @@ As an example, if SDL Core requests to change the layout to the `MEDIA` template
...
+              "characterSet":"TYPE2SET",
+              "width":500,
+              "rows":1
+            },
+            {
+              "name":"subtleAlertText2",
+              "characterSet":"TYPE2SET",
+              "width":500,
+              "rows":1
+            },
```
Comment (ShobhitAd) [inline docs/Integrating Your HMI/Developing the HMI UI/index.md:314]: Should `subtleAlertSoftButtonText` not also be included here?

Comment (jacobkeeler) [reply docs/Integrating Your HMI/Developing the HMI UI/index.md:314]:
```suggestion
            },
            {
              "name":"subtleAlertSoftButtonText",
              "characterSet":"TYPE2SET",
              "width":50,
              "rows":1
            },
```

**Comment by ShobhitAd** [comment]
> I think `subtle_alert.png` should be a screenshot of the whole hmi(like the one for UI.Alert) so the viewer can see where the subtle_alert appears in the hmi

This comment still needs to be resolved. If you agree, you could generate your own screenshot for the subtle alert or use one of these screenshots if you like
Screenshot from 2020-08-11 10-29-51
Screenshot from 2020-08-11 10-57-54

**Pull request merged**